app/mixins/video-record.js

```diff
@@ -129,6 +129,14 @@ export default Ember.Mixin.create({
      */
     recorderReady: false,
 
+    /**
+     * Upper limit on recording length in seconds. Used to enforce a hard limit on the
+     * user-defined maxRecordingLength value.
+     * @property {Number} recordingLengthUpperLimit
+     * @private
+     */
+    recordingLengthUpperLimit: 7200,
+
```

Comment:
FYI: I had to create a separate property to store this upper limit value because the default `maxRecordingLength` value is not accessible in the `setUpRecorder` code. This is because the `maxRecordingLength` value is overwritten by the researcher parameters by the time the `setUpRecorder` code runs and the default isn't stored anywhere else. 
This does make it possible for us to change this in the future so that the upper limit is higher than the default, which might make sense if there are some rare cases where researchers need longer recordings.

Reply:
Looks great! Good thinking on making this more flexible for the future.